### PR TITLE
chore: Exporting basic frontend telemetrics to new relic

### LINKED ISSUE #28445: Export basic Frontend Telemetrics to New Relic
Will be exporting basic frontend telemetrics from our release environment to new relic.

### PR DESCRIPTION
**Pull request by vsvamsi1**
## Description
Exporting release frontend tele-metrics to new relic.
#### PR fixes following issue(s)

Fixes #28445
#### Type of change
- Chore (housekeeping or task changes that don't impact user perception)
#### How Has This Been Tested?
- [x] Manual
- [ ] JUnit
- [ ] Jest
- [ ] Cypress
>
>
#### Dev activity
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] PR is being merged under a feature flag

#### QA activity:
- [ ] [Speedbreak features](https://github.com/appsmithorg/TestSmith/wiki/Guidelines-for-test-plans#speedbreakers-) have been covered
- [ ] Test plan covers all impacted features and [areas of interest](https://github.com/appsmithorg/TestSmith/wiki/Guidelines-for-test-plans#areas-of-interest-)
- [ ] Test plan has been peer reviewed by project stakeholders and other QA members
- [ ] Manually tested functionality on DP
- [ ] We had an implementation alignment call with stakeholders post QA Round 2
- [ ] Cypress test cases have been added and approved by SDET/manual QA
- [ ] Added `Test Plan Approved` label after Cypress tests were reviewed
- [ ] Added `Test Plan Approved` label after JUnit tests were reviewed

### CODE REVIEW

**Review by rajatagrawal — APPROVED** [review]

File: app/client/public/index.html [hunk 1]
```diff
@@ -17,30 +17,53 @@
...
+      const newRelicLicenseKey=parseConfig("__APPSMITH_APP_NEW_RELIC_LICENSE_KEY__");
 
-        return result;
-      };
-      const CLOUD_HOSTING = parseConfig("__APPSMITH_CLOUD_HOSTING__");
-      const ZIPY_KEY = parseConfig("__APPSMITH_ZIPY_SDK_KEY__");
-      const AIRGAPPED = parseConfig("__APPSMITH_AIRGAP_ENABLED__");
-    </script>
+      
+      ;window.NREUM||(NREUM={});NREUM.init={distributed_tracing:{enabled:true},privacy:{cookies_enabled:true},ajax:{deny_list:["bam.nr-data.net"]}};
```
Comment (SatishGandham) [inline app/client/public/index.html:53]: @vsvamsi1 , we should extract the new relic script to a separate file and load it only on cloud.

**Pull request merged**